basiclibrary/src/test/java/basiclibrary/LibraryTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basiclibrary;
+
+import basiclibrary.Library;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+
+public class LibraryTest {
+
+   @Test
+    public void testRollingDice() {
+     //check the length of the newArray
+     int[] array = Library.roll(3);
+     assertEquals("this is the result []", array.length, 3);
+     for (int i = 0; i < array.length; i++) {
```

Comment:
This should be its own test and only called once, rather than in multiple tests. I would make a function testRollsOnlyOneThroughSix for that